diff -ur udftools-1.0.0b3.orig/wrudf/wrudf.c udftools-1.0.0b3/wrudf/wrudf.c
--- udftools-1.0.0b3.orig/wrudf/wrudf.c	2002-11-26 08:18:51.000000000 +0100
+++ udftools-1.0.0b3/wrudf/wrudf.c	2005-08-17 02:06:33.000000000 +0200
@@ -245,7 +245,7 @@
 	    } else if( strncmp( spm->partIdent.ident, UDF_ID_VIRTUAL, strlen(UDF_ID_VIRTUAL)) == 0 )
 		virtualPartitionNum = i;
 	}
-	(char*)spm += spm->partitionMapLength;
+	spm = (char*)spm + spm->partitionMapLength;
     }
 
     if( medium == CDR ) {
