diff -Nur udftools-1.0.0b3.orig/cdrwtool/cdrwtool.c udftools-1.0.0b3/cdrwtool/cdrwtool.c
--- udftools-1.0.0b3.orig/cdrwtool/cdrwtool.c	2002-12-28 05:48:51.000000000 +0100
+++ udftools-1.0.0b3/cdrwtool/cdrwtool.c	2007-12-28 12:09:14.000000000 +0100
@@ -27,6 +27,7 @@
 #include <unistd.h>
 #include <getopt.h>
 #include <signal.h>
+#include <limits.h>
 
 #include <sys/ioctl.h>
 #include <asm/param.h>
diff -Nur udftools-1.0.0b3.orig/pktsetup/pktsetup.c udftools-1.0.0b3/pktsetup/pktsetup.c
--- udftools-1.0.0b3.orig/pktsetup/pktsetup.c	2002-11-26 08:18:51.000000000 +0100
+++ udftools-1.0.0b3/pktsetup/pktsetup.c	2007-12-28 12:09:30.000000000 +0100
@@ -21,6 +21,7 @@
 #include <sys/ioctl.h>
 #include <unistd.h>
 #include <getopt.h>
+#include <limits.h>
 #include <bits/types.h>
 #include <sys/types.h>
 
